triggers/tests/daml/ACS.daml

```diff
@@ -19,8 +19,8 @@ initState : Party -> ActiveContracts -> TriggerState
 initState party (ActiveContracts events) = TriggerState (foldl updateAcs TM.empty events) 0 party
   where
     updateAcs : TextMap Identifier -> Created -> TextMap Identifier
-    updateAcs acs (Created _ cId tId)
-      | tId.entityName == "Asset" = TM.insert cId tId acs
+    updateAcs acs (Created _ cId)
+      | cId.templateId.entityName == "Asset" = TM.insert cId.contractId cId.templateId acs
```

Comment:
I’m confused, it looks like you don’t expose `AnyContractId` (or more likely I’m too stupid to see it) but you can still access the fields?

Reply:
Indeed, that's why I didn't notice that I hadn't exposed it. I don't know how the field is accessible without export. The code still compiles, even if I explicitly export the type without the constructors.

Reply:
Oh I think that might be because of how our record desugaring (the dot syntax used here) works. That does seem rather confusing and potentially problematic since it removes the ability to have abstract record types.